### PR TITLE
Feature#29 crud companies and employee

### LINKED ISSUE #29: CRUD Companies and Employee
# REQUIREMENTS
- Must add functionality to the companies view
- Should be able to add, delete and modify a company in the database
- Must add functionality to the employees table view
- Should be able to add, disable and modify an employee in the database

### PR DESCRIPTION
**Pull request by DerekAyala**
# Description
What did you implemented/Why did you implemented this:
 - create crud operations in table companies and employees
 - validate uniques emails , employees id and private numbers
 - if a company is disabled , the employees will be disabled too
 - they are disabled not destroyed
 - change modals for complete views and the first design
 - a edit file for employees and companies.

  ## Screenshots
index company
index_company
new company
new_company
edit company 
edit_company
errors msg in company
errors_company

index employee
index_employee

new employee
new_employee

edit employee 
Edit_employee
errors msg in employee
errors_employee

### CODE REVIEW

File: app/assets/stylesheets/src/layout/table.css.scss [hunk 1]
```diff
@@ -38,18 +38,44 @@
...
-.modal-body{
+.resources{
   font-family: $font;
   font-size: 2rem;
   text-align: left;
-  width: 100%;
+  background-color: $background_table;
+  border: 0.1rem solid $secondary_color;
+  padding: 1rem;
+  width: 50rem;
```
Comment (JuanequeX) [inline app/assets/stylesheets/src/layout/table.css.scss]: Sort this block

File: app/assets/stylesheets/src/layout/table.css.scss [hunk 2]
```diff
@@ -38,18 +38,44 @@
...
+    gap:1rem;
     grid-template-columns: 1fr;
     .form__label{
-      color: black;
+      color: $font_color;
       padding: 1rem;
+      width: 100%;
     }
     .form__input{
       background-color: $background_table;
```
Comment (JuanequeX) [inline app/assets/stylesheets/src/layout/table.css.scss]: Modify this blocks with Bem nomeclature, please, here have a [references](https://andrew-barnes.medium.com/bem-and-sass-a-perfect-match-5e48d9bc3894)

File: app/controllers/companies_controller.rb [hunk 3]
```diff
@@ -1,4 +1,36 @@
 class CompaniesController < ApplicationController
   def index
+    @company = Company.where(disable: false)
+  end
```
Comment (JuanequeX) [inline app/controllers/companies_controller.rb:4]: Please add a extra line here

**Review by JuanequeX — CHANGES_REQUESTED** [review]

**Review by YilKanda — APPROVED** [review]
LGTM!

**Pull request merged**